spinnaker/copy_kube_config/remove_temp_user.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I don't think this script really makes sense, it's a one-liner. Do you plan to add extra commands in the future?